### DISCUSSION TITLE
Is it possible to let PnP Modern Search search also in other columns than title

### DISCUSSION THREAD

**Discussion by jbmueller0584** [microsoft-search/pnp-modern-search#4159] [Q&A]
Hi there,

we built a UI in SharePoint Online with PnP Modern Search v. 4.13.1

We use a custom HTML template to display everything on the page.

We mapped all the list columns to RefinableStrings.

We realized that only the title column is searched, which means when we search for values that are present in different columns than title the result wont affect these items.

Searchpage:
![SolutionSuche1](https://github.com/user-attachments/assets/ca1136a9-b8db-478c-8d6d-c981300cb4b8)

Corresponding list:
![SolutionSuche2](https://github.com/user-attachments/assets/99c13b09-c0c7-49a7-afa8-9cccd35c5faf)

e. g. searching for the SolutionID in Modern Search wont work.

Please let me know how it is possible to configure Modern Search in a way let it search in every column of that list.

Thank you in advance!

Great pre-christmas time you all 🙋‍♂️

**Comment by kasperbolarsen** [comment]
Are you talking about the scenario where a query like "Ready for review" returns no hits where RefinableStringXX:"Ready for review" returns all the items where the column "Review Status" is equal to "Ready for review"?

If the answer is yes, then the reason is that Refinable properties are not searchable,  so you will have to create a new managed property for each of the Refinables, mapped to the same crawled properties

**Comment by wobba** [comment]
Seems you're a victim of https://www.techmikael.com/2020/03/side-effect-of-mapping-crawled.html. If you want content searchable you have to map them to a searchable managed property as well.
